omod/src/main/java/org/openmrs/module/idgen/rest/resource/LogEntryResource.java

```diff
@@ -98,7 +98,7 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 
 	@PropertyGetter("source")
 	public String getSource(LogEntry logentry) {
-		return logentry.getSource().getUuid();
+		return logentry.getSource().getName();
```

Comment:
why did you change this?

Reply:
For usability, the Identifier Source names should be displayed and not the uuids

Reply:
if you remove the property getter for source, since you don't need it, the source property will return the whole source object that contains all the source fields such as name, Uuid etc

Reply:
I need to return just the names because I passing them as options to a select

Reply:
@dkayiwa what's your take on the above?

Reply:
Completely remove this proper PropertyGetter.
We use them for properties that are not part of the resource object. For instance LogEntry does not have a display property and hence why we need a PropertyGetter for it. But for source, we do not need it. Therefore remove it completely.